Tests/RoslynPluginGeneratorTests/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="FluentAssertions" version="5.4.1" targetFramework="net452" />
-  <package id="NuGet.CommandLine" version="3.3.0" targetFramework="net452" />
-  <package id="System.ValueTuple" version="4.4.0" targetFramework="net452" />
+  <package id="FluentAssertions" version="5.4.1" targetFramework="net46" />
+  <package id="NuGet.CommandLine" version="4.7.0" targetFramework="net46" developmentDependency="true" />
```

Comment:
Is it normal to have only this `packages.config` getting the `developmentDependency="true"` option?

Reply:
The _NuGet.CommandLine_ package is marked as a _developmentDependency_ so this will automatically be added whenever the package is referenced. However, in the case of the generator project, it's actually a runtime dependency, so I manually removed the _developmentDependency_ in that case.